src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -106,17 +106,19 @@
    * active class (the one that has a different colored border), we remove the extra padding
    * and set the border
    */
-  margin-right: 12px;
   padding: 2px 9px 3px;
   border: none;
   border-radius: var(--radius-normal);
   background-color: color-mod(var(--temp-grey-blue-7) alpha(10%));
-  font-size: 13px;
-  font-weight: var(--weight-medium);
   color: var(--temp-grey-blue-7);
   cursor: pointer;
 }
 
+.walletAddress span {
+  font-size: 13px;
+  font-weight: 600;
```

Comment:
Do we have a font that has 600 weight available? I think we only have 400, 500, and 700

Reply:
Thats true 🤔  Designs want font-weight: 600, i think i will switch it to 500 for now and we will see feedback on QA

Reply:
If there's any feedback on QA about this, ignore it for now.

We're not adding a new font-weight just for this. That involves adding the required font files, which are about 300k